app/Http/Middleware/Admin.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class Admin
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        return auth()->user()->role == 1 ? $next($request) : to_route('subscriber');
```

Comment:
Your roles are not fully readable. 

Ideally, you should've used a single char string or enum.